src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/virtualusers/Ec2VirtualUsersFormula.kt

```diff
@@ -110,6 +110,7 @@ class Ec2VirtualUsersFormula private constructor(
             browser = formula.browser
             network = formula.network
             nodeOrder = formula.nodeOrder
+            instanceType = formula.instanceType
```

Comment:
Ah, we've been hit with copying builders before: covered in 56f293cd5abdbbe7d762a1a32b864c3e68900e1e and fixed in 49be8499728d6b6e8bddcfcb6f0903fb1441437a, but not asserted.
Assertions of these internals are tricky in integration tests. It should be feasible in unit tests. But I won't require a TDD red-green-refactor commit structure here.